tpu-client-next/src/workers_cache.rs

```diff
@@ -149,6 +158,45 @@ impl WorkersCache {
         send_res
     }
 
+    /// Sends a batch of transactions to the worker for a given peer.
+    ///
+    /// If the worker for the peer is disconnected or fails, it
+    /// is removed from the cache.
+    pub async fn send_transactions_to_address(
```

Comment:
this adds back the code that have been deleted couple of PR ago, so not the new code.

Reply:
Moved this code to a separate small PR: https://github.com/anza-xyz/agave/pull/4522